apache-rat-core/pom.xml

```diff
@@ -44,20 +44,17 @@
           <artifactId>apache-rat-plugin</artifactId>
           <configuration>
             <inputExcludes>
-              <!-- remove if RAT-501 is fixed -->
+              <!-- exclude the definition file(s) -->
               <exclude>**/default.xml</exclude>
               <exclude>**/example-configuration.xml</exclude>
```

Comment:
on master I added license headers to these files.

Reply:
Not sure if it is a regression as these files were not marked in releases <0.17. Maybe it was introduced to the change to use Tika for document type detection instead of the proprietary checks in RAT<0.17 and is not a bug.

Thanks for opening RAT-502 concerning the XSLT files.